src/librekey/rnp_key_store.c

```diff
@@ -856,3 +924,97 @@ rnp_key_store_get_next_key_by_name(pgp_io_t *             io,
 {
     return get_key_by_name(io, keyring, name, n, key);
 }
+
+static bool
+grip_hash_bignum(pgp_hash_t *hash, const BIGNUM *bignum)
+{
+    uint8_t *bn;
+    size_t   len;
+    int      padbyte;
+
+    if (BN_is_zero(bignum)) {
+        pgp_hash_add(hash, (const uint8_t *) &"\0", 1);
+        return true;
+    }
+    if ((len = (size_t) BN_num_bytes(bignum)) < 1) {
```

Comment:
Is it really possible to happen? Above you check if it is 0, so here it must have at least 1 bit, so 1 byte.